app/src/server.js

```diff
@@ -6,45 +6,50 @@
  * Main entry point for the Digital Citizenship proxy.
  */
 
-require("dotenv").load();
-
+import tokenStrategy from "./strategies/tokenStrategy";
+import spidStrategy from "./strategies/spidStrategy";
 import container from "./container";
-import type { SessionStorageInterface } from "./services/sessionStorageInterface";
 import ProfileController from "./controllers/profileController";
 import AuthenticationController from "./controllers/authenticationController";
 
+require("dotenv").load();
+
+const bodyParser = require("body-parser");
+const fs = require("fs");
 const express = require("express");
+const https = require("https");
 const morgan = require("morgan");
 const passport = require("passport");
-const Strategy = require("passport-http-bearer");
-
-const port = process.env.PORT || 8080;
-
-passport.use(
-  new Strategy(function(token, done) {
-    const sessionStorage = (container.resolve(
-      "sessionStorage"
-    ): SessionStorageInterface);
-    const user = sessionStorage.get(token);
-    if (user) {
-      return done(null, user);
-    } else {
-      return done(null, false);
-    }
-  })
-);
+
+const port = process.env.PORT || 443;
+
+// Setup Passport.
+
+passport.use(tokenStrategy);
```

Comment:
pls add a comment to each line describing briefly what each passport and express plugin/middleware does